openverse_catalog/docs/data_models.md

```diff
@@ -0,0 +1,57 @@
+TODO: This documentation is temporary and should be replaced by more thorough documentation of our DB fields in https://github.com/WordPress/openverse-catalog/issues/783>
+
+# Data Models
+
+The following is temporary, limited documentation of the columns for each of our Catalog data models.
+
+## Required Fields
+
+| field name | description |
+| --- | --- |
+| *foreign_landing_url* | URL of page where the record lives on the source website. |
+| *audio_url* / *image_url* | Direct link to the media file. Note that the field name differs depending on media type. |
+| *license_info* | LicenseInfo object that has (1) the URL of the license for the record, (2) string representation of the license, (3) version of the license, (4) raw license URL that was by provider, if different from canonical URL |
+
+## Optional Fields
+
+The following fields are optional, but it is highly encouraged to populate as much data as possible:
+
+| field name | description |
+| --- | --- |
+| *foreign_identifier* | Unique identifier for the record on the source site. |
```

Comment:
I really wondered about this! Obviously most of our provider scripts are already returning `None` early if the foreign id is not populated, but not all of them (Europeana doesn't), and none of the media stores do any validation. The table itself appears to `default uuid_generate_v4()`.

I ended up leaving it off because previous docs excluded it from the required fields as well 🤷‍♀️ I think it makes sense though to add it as required for _future_ providers, and we can decide if there are any more action items in the data normalization project.